src/components/MoviesList/MoviesList.tsx

```diff
@@ -1,6 +1,18 @@
 import React from 'react';
 import './MoviesList.scss';
+import { Movie } from '../../types/Movie';
+import { MovieCard } from '../MovieCard';
 
-export const MoviesList: React.FC = () => (
-  <>Put the list here</>
+type Props = {
+  movies: Movie[]
+};
+
+export const MoviesList: React.FC<Props> = ({ movies = [] }) => (
+  <div className="movies">
+    {movies.map((oneMovie: Movie) => (
+      <MovieCard
+        movie={oneMovie}
+      />
+    ))}
+  </div>
```

Comment:
Let's use semantic tags for the list and it's items

Also you should pass `key` attribute when you want to render array items